lib/index.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <meta name="app-build-reference" content="<%= htmlWebpackPlugin.options[ 'build-reference' ] %>">
+    <meta name="node-version" content="<%= htmlWebpackPlugin.options[ 'node-version' ] %>">
+    <meta name="build-platform" content="<%= htmlWebpackPlugin.options[ 'build-platform' ] %>">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300italic,400,400italic,600,600italic" />
```

Comment:
@drw158 are we using Source Sans Pro anymore? Can we remove this reference? It's a pretty hefty network payload.